src/site/content/en/fast/image-cdns/index.md

```diff
@@ -0,0 +1,103 @@
+
+
+### Transformations
+
+Image CDNs offer tens, and in some cases hundreds, of different image transformations. These transformations are specified via the URL string and there are no restrictions on using multiple transformations at the same time. In the context of web performance, the most important image transformations are size, pixel density, format, and compression. These transformations are the reason why switching to an image CDN typically results in a significant reduction in image size.
+
+There tends to be an objectively “best” setting for performance transformations, so some image CDNs support an “auto” mode for these transformations. For example, instead of `format=webp` you would include `format=auto` in the image’s URL string. This setting would leave it up to the CDN to determine the best format to serve to a client. Signals that an image CDN can use to determine the best way to transform an image include:
+
+
+
+*   [Client hints](https://developers.google.com/web/updates/2015/09/automating-resource-selection-with-client-hints) (e.g. Viewport-Width, DPR, and Width)
+*   [Save-Data](https://developer.mozilla.org/en-US/docs/Web/HTTP/Headers/Save-Data) header
+*   [user-agent](https://developer.mozilla.org/en-US/docs/Web/HTTP/Headers/User-Agent) header
+*   [NetworkInformation API](https://developer.mozilla.org/en-US/docs/Web/API/Network_Information_API)
+
+For example, the image CDN might serve JPEG XR to an Edge browser, WebP to a Chrome browser, and JPEG to a very old browser. “Auto” settings are popular because they allow you to take advantage of image CDNs’ significant expertise in optimizing images without you needing to make code changes in order to adopt new technologies once they’re supported by the image CDN.
+
+
+## Types of Image CDNs
+
+Image CDNs can be broken down into two categories: self-managed and third-party-managed.
+
+
+### Self-managed image CDNs
+
+Self-managed CDNs can be a good choice for sites with engineering personnel that are comfortable maintaining their own infrastructure.
+
+[Thumbor](https://github.com/thumbor/thumbor) is the only self-managed image CDN available today. While it is open-source and free to use, it generally has less features than most commercial CDNs and its documentation is somewhat limited. [Wikipedia](https://wikitech.wikimedia.org/wiki/Thumbor), [Square](https://medium.com/square-corner-blog/dynamic-images-with-thumbor-a430a1cfcd87), and [99designs](https://99designs.com/tech-blog/blog/2013/07/01/thumbnailing-with-thumbor/) are three sites that use Thumbor. See [How to install the Thumbor image CDN](https://web.dev/install-thumbor) for instructions on setting it up.
```

Comment:
```suggestion
[Thumbor](https://github.com/thumbor/thumbor) is the only self-managed image CDN available today. While it is open-source and free to use, it generally has fewer features than most commercial CDNs, and its documentation is somewhat limited. [Wikipedia](https://wikitech.wikimedia.org/wiki/Thumbor), [Square](https://medium.com/square-corner-blog/dynamic-images-with-thumbor-a430a1cfcd87), and [99designs](https://99designs.com/tech-blog/blog/2013/07/01/thumbnailing-with-thumbor/) are three sites that use Thumbor. See the [How to install the Thumbor image CDN](https://web.dev/install-thumbor) post for instructions on setting it up.
```
Edits:
- "Less" is for non-countable nouns (like "water"), and "fewer" is for countable nouns (like "features").
- Need a comma between independent clauses joined by a coordinating conjunction.
- Our convention for referring to posts is "the [title] post."